builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -208,9 +228,17 @@ node()  {
 				}
 				events.sendStartedEvent(asset_event, "${asset_event_message} assets starts", utilModule.generateAssetMap(service_config['provider'], "arn:aws:s3:::${asset_info}/*", "s3", service_config))
 				try {
-					sh "aws s3 sync ./app s3://${asset_info}  --exclude \".git/*\" --exclude \".gitignore\" --exclude \"*.svg\" --profile ${credsId}"
-					sh "aws s3 sync ./app s3://${asset_info}  --exclude \"*\" --include \"*.svg\" --no-guess-mime-type --content-type image/svg+xml --profile ${credsId}"
-					events.sendCompletedEvent(asset_event, null, utilModule.generateAssetMap(service_config['provider'], "arn:aws:s3:::${asset_info}/*", "s3", service_config), environment);
+					if(service_config['framework'] == 'angular') {
```

Comment:
@pspani Here in the following scripts, profile tag is missing. Isn't it expected?

Reply:
its exected, added them